coturn/coturn/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  gateway:
+    build: gateway
+    ports:
+      - "8000:8000"
+    hostname: "kaazing.example.com"
```

Comment:
I see the misunderstanding is due to something I missed in the original PR: https://github.com/kaazing/gateway/pull/674/files#r74501870